.bazelci/presubmit.yml

```diff
@@ -29,10 +29,8 @@ tasks:
       - "--" # Allows negative patterns; hack for https://github.com/bazelbuild/continuous-integration/pull/245
       - "..."
       - "@examples//..."
-      # Skip tests for dylib support on osx, since we don't support it yet.
+      # This test fails under bazel test but not when run directly due to -rpath issues.
```

Comment:
Can you make an issue that shows the runtime error and link that here so it's easier to keep track of? 🙏

Reply:
This is related to this comment https://github.com/bazelbuild/rules_rust/issues/621#issuecomment-832382072. I'm trying to implement the change to generate rpath properly but it will still not work until we use a version of bazel that responds to `--incompatible_macos_set_install_name`